### PR TITLE
11 issue

### LINKED ISSUE #11: Create a GET route to get the assessments
We will need to be able to feed our Assessment Content to the client-side of our application.

Create a GET route which feeds this content via our API, something like /assessments could work.

Refer to the [wiki](https://github.com/harakeke-2019/EDAMAME/wiki/Server-Routes) to see how the routes may need to be setup.

### PR DESCRIPTION
**Pull request by Jennyfang135**
closes #11 issue.
I added server side route (assessmentsRoute) for getting the assessments, I also created a n assessmentDb.js file for retrieving assessments data from database.

### CODE REVIEW

File: server/routes/starter.js [hunk 1]
```diff
@@ -6,4 +6,5 @@ router.get('/', (req, res) => {
   res.send('Welcome to the start of EDAMAME!')
 })
 
+
```
Comment (bryceknz) [inline server/routes/starter.js:9]: Why is this newline added?

File: tests/assessments.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const server = require('../server/routes')
```
Comment (bryceknz) [inline tests/assessments.test.js]: This test is failing.

File: server/routes/assessmentsRoute.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const assessmentsRouter = express.Router()
+const assessmentsDb = require('./assessmentsDb')
```
Comment (bryceknz) [inline server/routes/assessmentsRoute.js]: This require fails, I think the filepath may be incorrect.

File: server/routes/assessmentsRoute.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+router.get('/:id', (req, res) => {
```
Comment (bryceknz) [inline server/routes/assessmentsRoute.js]: I have a sneaking suspicion this code won't work as `router` isn't defined anywhere.

File: tests/assessments.test.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    .expect('content-type', 'text/html; charset=utf-8')
+    .end((err, res) => {
+      expect(err).toBeNull()
+
+      const $ = cheerio.load(res.text)
+      const actual = $('h1').text()
+
+      console.log(actual)
+
+      expect(actual).toMatch('Turing')
```
Comment (bryceknz) [inline tests/assessments.test.js]: Is this testing what you would expect to come back from the `assessments/:id` GET route?

**Review by bryceknz — CHANGES_REQUESTED** [review]
Please see the suggested changes.

File: server/db/assessmentsDb.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+const environment = 'development'
```
Comment (bryceknz) [inline server/db/assessmentsDb.js]: Please utilise the database config already built into `db/index.js`. This is to avoid duplicated code as we will need the same config in each new db module.

File: server/db/cohorts.js [hunk 7]
```diff
@@ -1,4 +1,7 @@
-const connection = require('./')
+const environment = 'development'
```
Comment (bryceknz) [inline server/db/cohorts.js]: Please utilise the database config already built into `db/index.js`. This is to avoid duplicated code as we will need the same config in each new db module.

File: server/routes/signin.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
+const express = require('express')
```
Comment (bryceknz) [inline server/routes/signin.js]: This code doesn't seem to be related to issue #11 ?

File: server/server.js [hunk 9]
```diff
@@ -4,11 +4,14 @@ const express = require('express')
 const server = express()
 
 const starterRoute = require('./routes/starter')
-const assessmentsRoute = require ('./routes/assessmentsRoute')
+const cohortRoutes = require('./routes/cohortRoutes')
```
Comment (bryceknz) [inline server/server.js:8]: Why is `cohortRoutes` being added? This is unrelated to issue #11 .

**Review by bryceknz — CHANGES_REQUESTED** [review]

File: server/server.js (hunk 9 above)
Comment (Jennyfang135) [reply server/server.js:8]: Hi, Bryce, that's part of conflict resolving when I pull from dev branch, it was added on.

**Review by bryceknz — APPROVED** [review]
